packages/framework-integration-tests/integration/helper/sleep.ts

```diff
@@ -1,3 +1,31 @@
 export async function sleep(ms: number): Promise<void> {
   return new Promise((resolve) => setTimeout(resolve, ms))
 }
+
+export async function waitForIt<TResult>(
```

Comment:
No logic change, just moved it out of the provider-specific folder so that I can use it in end-to-end tests